build.gradle

```diff
@@ -1,3 +1,9 @@
+buildscript {
+    dependencies {
+        classpath 'com.google.code.gson:gson:2.10.1'
+    }
+}
+
```

Comment:
What is this needed for?

Reply:
this was a stupid bug with *something* pulling in the wrong version of gson and breaking everything, and this was how I fixed it (this was in Hexal; plausibly it wasn't going to be an issue here, I just copied all the changes I made in Hexal straight here)

Reply:
Haven't seen that issue building locally; will re-add if we run into it here